docs/source/getting_started/welcome_tour.rst

```diff
@@ -112,11 +112,21 @@ simple:
     class MixUp(Algorithm):
         def match(self, event: Event, state: State) -> bool:
             """Determine whether the algorithm should run on a given event."""
-            return event == "after_dataloader"
+            return event in [Event.AFTER_DATALOADER, Event.AFTER_LOSS]
 
         def apply(self, event: Event, state: State, logger: Logger) -> None:
             """Run the algorithm by modifying the State."""
-            state.inputs, state.targets = mixup_batch(state.inputs, state.targets)
+            input, target = state.batch_pair
+
+            if event == Event.AFTER_DATALOADER:
+                new_input, _, self.permuted_target, self.mixing = mixup_batch(input, target, alpha=0.2)
+                state.batch = (new_input, target)
+
+            if event == Event.AFTER_LOSS:
+                modified_batch = (input, self.permuted_target)
+                new_loss = state.model.loss(state.outputs, modified_batch)
+                state.loss *= (1 - self.mixing)
+                state.loss += self.mixing * new_loss
```

Comment:
can this be one line? Took me a second to see that this was just a convex combo with it split on two lines with in-place operators.

Reply:
Switched to doing it in one line.